sql/moz-fx-data-shared-prod/firefox_accounts/fxa_all_events/view.sql

```diff
@@ -136,11 +146,13 @@ unioned AS (
 SELECT
   `timestamp`,
   receiveTimestamp,
+  time,
```

Comment:
I'm a little leery of adding a `schema.yaml` file for this view since the schema will constantly be evolving.

Maybe we can just give the `time` field a more intuitive name?  However, I don't have the context to come up with an appropriate alternative name.

@biancadanforth can you clarify what exactly the `time` field being pulled from `jsonPayload.fields` represents?

Reply:
I took a look at an example event with these timestamps[1]:
```json
{
  "timestamp": "2023-06-14 02:43:47.248954 UTC",
  "receiveTimestamp": "2023-06-14 02:43:51.319930 UTC",
  "time": "1686710626008.0"
}
```

All 3 timestamps are within 5 seconds of each other.
* `time` is the time the "Amplitude" event occured (in ms since the epoch).
  * For the subscription creation funnel events for example, it's the time we make the request to the `POST /metrics` endpoint to log the event.
* `timestamp` is likely the time that [`mozlog` logged the statement](https://github.com/mozilla/mozlog/blob/1b9cf8dd3a2a681c0d4aefbe76053c43ed52760e/lib/formatters/heka.js#L70).
* `receivedTimestamp` is likely [the time BigQuery received the log line](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry)[2].

I would be in favor of keeping `time`, but open to renaming it to something more descriptive.

---

[1] SQL query in FxA logs:
```sql
SELECT
  timestamp, receiveTimestamp, jsonPayload.fields.time as time
FROM
  `moz-fx-fxa-prod-0712.fxa_prod_logs.stdout`
WHERE
  DATE(timestamp) = "2023-06-14"
  AND jsonPayload.fields.event_type='fxa_pay_setup - success'
LIMIT
  10
```

[2]: We are using BigQuery and not Cloud Logging here, but I imagine it's similar.

Reply:
> All 3 timestamps are within 5 seconds of each other.

They're usually pretty close, but sometimes they vary more.  Last week when I checked a random day (2023-06-08) the biggest variance was `timestamp` & `receiveTimestamp` being 90 minutes after `time`.

> `timestamp` is likely the time that [`mozlog` logged the statement](https://github.com/mozilla/mozlog/blob/1b9cf8dd3a2a681c0d4aefbe76053c43ed52760e/lib/formatters/heka.js#L70).

Thanks for that code reference.  IMO it'd make most sense for `mozlog` to use `time` as the basis for the `timestamp` value it sends, but maybe there are use cases/edge cases where that wouldn't be desirable.

> `receivedTimestamp` is likely [the time BigQuery received the log line](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry)[2].
> [2]: We are using BigQuery and not Cloud Logging here, but I imagine it's similar.

I think it's when Cloud Logging received the log line as documented in the reference you linked. AFAIK the Cloud Logging BigQuery sink is just passing through the Cloud Logging values as is.

> I would be in favor of keeping `time`, but open to renaming it to something more descriptive.

I agree.  How about `event_time`?

@akkomar do you think naming it `event_time` would sufficiently disambiguate this from the other timestamps?

Reply:
`event_time` sounds good to me!

Reply:
I'll go with `event_time` as the consensus.

@kik-kik good point about this view's schema being more stable, though I'm not sure if `schema.yaml` files for views work either.

> `timestamp` is likely the time that [`mozlog` logged the statement](https://github.com/mozilla/mozlog/blob/1b9cf8dd3a2a681c0d4aefbe76053c43ed52760e/lib/formatters/heka.js#L70).

In that `mozlog` code it's setting `Timestamp` based on `record.timestamp`, and the FxA log tables have a `jsonPayload.timestamp` column which seems like it might be correlated.  I checked `jsonPayload.timestamp` and it's consistently about the same value as the top-level `timestamp` column (i.e. off from `jsonPayload.fields.time` by approximately the same amount).  So I guess the question is, how is the `record.timestamp` / `jsonPayload.timestamp` value originally being set? (not something we need to answer for this PR though)